spec/acceptance/qpid_tool_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper_acceptance'
+
+describe 'qpid::tools' do
```

Comment:
@wbclark Take a close look, this test appears to support that the issue in #46 is no longer an issue.

Reply:
Thanks for implementing this :+1: